e2e/test_operator/operator_test.go

```diff
@@ -1758,7 +1758,7 @@ var _ = Describe("Operator", Label("e2e", "pr"), func() {
 						fdbCluster.GetCluster().Status.ProcessGroups,
 						processGroupID,
 					)
-				}).WithTimeout(5 * time.Minute).WithPolling(5 * time.Second).Should(BeNil())
+				}).WithTimeout(10 * time.Minute).WithPolling(5 * time.Second).Should(BeNil())
```

Comment:
I increased the wait time here. In cases where the global sync mode is enabled, the coordination takes a bit longer, so the actual replacement often takes a bit longer than 5 min (from the last 2 failed tests it's often around 6min)